omod/pom.xml

```diff
@@ -34,13 +34,13 @@
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>webservices.rest-omod</artifactId>
-			<version>2.0</version>
+			<version>2.13</version>
```

Comment:
Use a property for this instead of repeating it.

Reply:
I've changed it.